README.md

```diff
@@ -89,41 +86,51 @@ If you want to send and receive messages to the embedded element you need to cal
 
 ## Building URLs for the SDK
 
+This section does not apply to cookieless embed as building URLs in this manner is not supported. See the [cookieless embed](#cookieless) section for details.
+
 The main documentation for Looker SSO embed URLs is [here](https://docs.looker.com/r/sdk/sso-embed). The only difference when creating URLs for the SDK is that you will need to add an `sdk=2` parameter to the Embed URL alongside other parameters like filters and the `embed_domain` parameter. This parameter allows Looker to identify that the SDK is present and can take advantage of additional features provided by the SDK.
 
 ```html
-/embed/looks/4?embed_domain=https://mywebsite.com => /embed/looks/4?embed_domain=https://mywebsite.com&sdk=2
+/embed/looks/4?embed_domain=https://mywebsite.com =>
+/embed/looks/4?embed_domain=https://mywebsite.com&sdk=2
 ```
 
 The SDK cannot add this parameter itself because it part of the signed SSO URL.
 
 ## The Auth Endpoint
 
+This section does not apply to cookieless embed as an alternate mechanism for authentication is used. See the [cookieless embed](#cookieless) section for details.
+
 In order to use the embed SDK on the frontend you must supply a backend service that handles authentication. This service is called by the SDK to generate a signed iframe URL that is unique to the requesting user. The backend process can either generate the signed embed URL itself using an embed secret or the backend process can generate the URL by calling the Looker API. Manual URL generation and signing avoids calling the Looker API resulting in decreased latency. Calling the Looker API requires less code and can be easier to maintain.
 
 ### Backend Process
-The *backend* process entails hosting a service at an endpoint such as `/auth` which does the following:
+
+This section does not apply to cookieless embed but a backend process is still required. See the [cookieless embed](#cookieless) section for details.
+
+The _backend_ process entails hosting a service at an endpoint such as `/auth` which does the following:
 
 1. The backend service initializes the [Looker API SDK](https://docs.looker.com/reference/api-and-integration/api-sdk) based on a client API key and secret typically stored in `Looker.ini` file.
```

Comment:
@kenvlooker is there a direct link to the new docs home we should use instead here?

Reply:
Good catch, we could update the link to this URL: https://cloud.google.com/looker/docs/api-sdk

Reply:
That said, the existing redirect is working.

Reply:
Updated